centreon/src/Core/ServiceCategory/Infrastructure/Repository/DbReadServiceCategoryRepository.php

```diff
@@ -489,33 +489,182 @@ private function createServiceCategoryFromArray(array $result): ServiceCategory
     }
 
     /**
-     * @param SqlConcatenator $concatenator
+     * @param int[] $accessGroupIds
+     *
+     * @return array<SqlConcatenator>
+     */
+    private function findServiceCategoriesRequest(array $accessGroupIds = []): array
```

Comment:
maybe use a more convenient name, while reading I expect to return an array of ServiceCategories or a SQL Request  to find the service categories (as a string), but not an array of WHERE condition concatenate

Reply:
for the naming I followed the same methodolody used for FindServices endpoint where `findServicesRequest` method returns a concatenator